README.md

```diff
@@ -10,14 +10,123 @@
+
+* log – within the log directory you will discover the application logs. This can be helpful for debugging purposes, but for production application I will typically use a outside service since they can offer more advanced services such as querying and alerts.
+
+* public – this directory contains some of the custom error pages, such as 404 errors, along with the robots.txt file which will let developers control how search engines index the application on the web.
+
+* test – if you did not bypass the Minitest installation, Rails will install the test directory; if you added the option of –skip-test-unit this test directory will not be included.
+
+* tmp – this is where the temporary items are stored and is rarely accessed by developers.
+
+* vendor – in earlier versions of Rails the public folder used to be where developers could place assets, such as CSS styles and JavaScript code, but in Rails 4+ the Rails asset pipeline responsibilities were moved to the app/assets directory. In Rails 4+ the vendor directory can be utilized to manage outside JavaScript frameworks.
+
+* Gemfile – the Gemfile contains all of the gems that are included in the application; this is where you will place outside libraries that are utilized in the application. After any change to the Gemfile you will need to run: bundle. This will call in all of the code dependencies into the application. The Gem process can seem like a mystery to new developers, but it is important to realize that the Gems that are brought into an application are simply Ruby files that help extend the functionality of the app.
+
+* Gemfile.lock – this file should not be edited; it displays all of the dependencies that each of the Gems contain along with their associated versions. There have been times where I discovered application bugs due to varying Gem’s dependencies.
+
+* README.rdoc – the readme file is an important place to document the details of the application. If the application is an open-source project, this is where you can place instructions to other developers, such as how to get the app up and running locally.
+
+## Creating the database
+
+Before we can startup the rails server, first you can create the database by running:
+
+```rake db:create```
+
+## Starting up the Rails Server
+
+To startup the Rails server, make sure that you are in the root of the application in the terminal and run:
+
+```rails s```
+
+This will startup the rails server and you will see output such as the following:
+
+```
+=> Booting WEBrick
+=> Rails 4.2.3 application starting in development on http://localhost:3000
+=> Run `rails server -h` for more startup options
+=> Ctrl-C to shutdown server
+[2015-11-14 22:16:54] INFO  WEBrick 1.3.1
+[2015-11-14 22:16:54] INFO  ruby 2.1.2 (2014-05-08) [x86_64-darwin13.0]
```

Comment:
Not important for this output, but we are generally running stuff through ruby 2.2.3